models/Applications.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const ApplicationsSchema = new Schema(
+  {
+    id: { type: mongoose.Schema.Types.ObjectId },
+    positions: { type: String, required: true, ref: 'Id Positions' },
```

Comment:
`ref` should be named as the collection's model. (Positions and Postulants)

Reply:
positions: { type: String, required: true, ref: 'Positions' }, ???

Reply:
exactly!

Reply:
Done!!